app/src/components/widgets/mostTimeConsumingTestCases/mostTimeConsumingTooltip/mostTimeConsumingTooltip.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import classNames from 'classnames/bind';
+import ReactDOMServer from 'react-dom/server';
+import Parser from 'html-react-parser';
+import { DATETIME_FORMAT_TOOLTIP } from 'common/constants/timeDateFormat';
+import TimeIcon from 'common/img/time-icon-inline.svg';
+import { TooltipWrapper } from '../../charts/common/tooltip/tooltipWrapper/index';
+import styles from './mostTimeConsumingTooltip.scss';
+
+const cx = classNames.bind(styles);
+
+export const MostTimeConsumingTooltip = (data) => (d) => {
```

Comment:
This lools like a component, but it's not. In this case it's a factory that creates function which renders static markup.
I'd better split this into a tooltip component and a render function which renders this tooltip to a static markup.
Or just renamed this to something like `createMostTimeConsumingTooltipRenderer`.